### ISSUE TITLE
Dread Spikes

### ISSUE THREAD

**Issue by Selindrile** [smd111/Gaze_check-windower-addon#4]
I went to manually add Dread Spikes to this (for obvious reasons) but realized as it's a spell not a monster_ability in resources it would need special handling, is that something you'd be willing/want to do?

**Comment by KenshiDRK** [comment]
There are a few problems for it: 1.You don't have a reliably way of knowing when the spikes are off 2.If you are a pld with aegis you don't want to turn on this (this could be solved by just adding a settings to turn on off). I'll let smd111 decide as its his addon. I have been thinking if I should add Pain Sync as well.

**Comment by smd111** [comment]
i dont mind if any thing is added as long as it does not bork anything that works and these things can be turned off

@Selindrile
however Dread Spikes is a buff and ever thing about it is calculated server side and there is no model change data client side for this (as far as i know)

@KenshiDRK
Pain Sync could be added if it gives model change data go ahead and add it
--you do have the rites to change the repo with out my consent


@both
if the ability does not change the model data of a mob this tool is not designed for that
just remember the core of this addon cant be changed or the it might cost to much to run it on some pc's that can run ffxi

**Comment by Selindrile** [comment]
Yeah this request isn't helpful really for people truly multiboxing, this was something I wanted to add simply because I sometimes miss the monster casting the spell, and die in one attack round (because Blue Mage), I can turn myself back around when ready. I would've just set the spell "Dread Spikes" itself as a perm_gaze and a long timer, except, as it's a spell it wouldn't be handled the same way in the resources.

If it's not something you're willing to add, no worries, I'll just do it the way I had my auto-turn set up previously in gearswap before I discovered your addon which handled it better.

**Comment by KenshiDRK** [comment]
Pain Sync doesn't need a model change, Pain Sync deals dmg based on the dmg you do to the mob while it is reading the move, thats why I was thinking to add it but the problem is there are ppl that use sentinel scherzo, earthen armor and migawari to counter it and that ppl won't want to turn.

**Comment by Professor4201** [comment]
Sorry to rehash this but was Dread Spikes ever added into this program? So many monsters use it now and if multi boxing deff gets my Blu killed haha.
